docs/reference-guides/core-blocks.md

```diff
@@ -340,7 +339,7 @@ Insert an image to make a visual statement. ([Source](https://github.com/WordPre
 -	**Name:** core/image
 -	**Category:** media
 -	**Supports:** anchor, color (~~background~~, ~~text~~), filter (duotone)
--	**Attributes:** align, alt, aspectRatio, caption, height, href, id, linkClass, linkDestination, linkTarget, rel, scale, sizeSlug, title, url, width
+-	**Attributes:** align, alt, aspectRatio, behaviors, caption, height, href, id, lightbox, linkClass, linkDestination, linkTarget, rel, scale, sizeSlug, title, url, width
```

Comment:
If `behaviors` is being deprecated, it shouldn't appear in the list of attributes. The migration should be handled by a [block deprecation](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-deprecation.md)

Reply:
Artemio has added the block depreciation in 79dad34fabcb526d268fc27d25aef037a8a594b7